mp4parse/src/lib.rs

```diff
@@ -2203,6 +2213,19 @@ fn read_frma<T: Read>(src: &mut BMFFBox<T>) -> Result<String> {
     String::from_utf8(code_name).map_err(From::from)
 }
 
+fn read_schm<T: Read>(src: &mut BMFFBox<T>) -> Result<SchemeTypeBox> {
+    // Flags can be used to signal presence of URI in the box, but we don't
+    // use the URI so don't bother storing the flags.
+    let (_, _) = read_fullbox_extra(src)?;
+    let scheme_type =  FourCC::from(be_u32(src)?);
+    let scheme_version = be_u32(src)?;
+    // Null terminated scheme URI may follow, but we don't use it right now.
```

Comment:
Even if we don't need the scheme_uri, we need to skip the rest of the box contents or we'll trigger a parser desync if it's present.  `skip_box_remain` will take care of that.